README.md

```diff
@@ -103,13 +103,13 @@ A list of configuration options which you need
 
     required configuations:
 
-    - `AWS_S3_ACCESS_KEY_ID`: identity
-    - `AWS_S3_SECRET_ACCESS_KEY`: password to authorize identity
-    - `AWS_S3_ENDPOINT_URL`: the url of the service
-    - `AWS_STORAGE_BUCKET_NAME`: the bucket name of the storage to access objects in path notation (not subdomain)
+    - `ALEXANDRIA_S3_ACCESS_KEY`: identity
+    - `ALEXANDRIA_S3_SECRET_KEY`: password to authorize identity
+    - `ALEXANDRIA_S3_ENDPOINT_URL`: the url of the service
+    - `ALEXANDRIA_S3_BUCKET_NAME`: the bucket name of the storage to access objects in path notation (not subdomain)
```

Comment:
I think you cannot change those variable names starting with `AWS_S3_` at will because the s3 storages backend is configured by those in the dependency code

Reply:
And thinking of that: since those env vars are dependent on which backend is used it would actually make sense to move those to `alexandria.storages.backend.<some_backend>` and load them only when said backend is selected so they would not clutter the alexandria settings proper.

Reply:
> I think you cannot change those variable names starting with AWS_S3_ at will because the s3 storages backend is configured by those in the dependency code

Yes, we can because we explicitly read those settings in the storage class: https://github.com/projectcaluma/alexandria/blob/bb8ad5a16beee65212c944b25777aeea0e30b35a/alexandria/storages/backends/s3.py#L7-L19

> And thinking of that: since those env vars are dependent on which backend is used it would actually make sense to move those to alexandria.storages.backend.<some_backend> and load them only when said backend is selected so they would not clutter the alexandria settings proper.

Not sure it's worth the effort to properly load env vars from the storage class file as we'd have problems if this is used as a django app (proper handing of `.env` files etc. which is dependent on the host app). But what we could do is configure the whole storage in an object kind of like the settings in rest framework. Something like:

```python
ALEXANDRIA_STORAGE = {
    "BACKEND": env.str("ALEXANDRIA_STORAGE_BACKEND", default="alexandria.storage.backends.s3.S3Storage")
    "S3_OPTIONS": {
        "BUCKET_NAME":  "..."
    }
}
```

Reply:
Haha, looks like I failed to reexamine the diff of the diff etc. after your force push from 7390da3 to 3905881

so apparently we agree.